internal/core/context.go

```diff
@@ -19,40 +19,32 @@ type meta struct {
 	BuildInfo *BuildInfo
 	Client    *scw.Client
 	Printer   printer.Printer
+	Commands  *Commands
 
-	stdout io.Writer
-	stderr io.Writer
+	runCommand *Command
```

Comment:
`ranCommand`?

Reply:
This refers to the command that ran in the Run function ==> Run command